frontend/src/pages/TrainingReports/components/EventCard.js

```diff
@@ -80,77 +100,93 @@ function EventCard({
   const contextMenuLabel = `Actions for event ${event.id}`;
 
   return (
-    <article
-      className="ttahub-event-card usa-card padding-3 radius-lg border width-full maxw-full smart-hub-border-base-lighter margin-bottom-2"
-      data-testid="eventCard"
-    >
-      <div className="ttahub-event-card__row position-relative">
-        <div className="ttahub-event-card__event-column ttahub-event-card__event-column__title padding-right-3">
-          <p className="usa-prose text-bold margin-y-0">Event title</p>
-          <p className="usa-prose margin-y-0">{data.eventName}</p>
-        </div>
-        <div className="ttahub-event-card__event-column ttahub-event-card__event-column__id padding-right-3">
-          <p className="usa-prose text-bold margin-y-0">Event ID</p>
-          <p className="usa-prose margin-y-0">
-            <Link to={link}>
-              {data.eventId}
-            </Link>
-          </p>
-        </div>
-        <div className="ttahub-event-card__event-column ttahub-event-card__event-column__organizer padding-right-3">
-          <p className="usa-prose text-bold margin-y-0">Event organizer</p>
-          <p className="usa-prose margin-y-0">{data.eventOrganizer}</p>
+    <>
+      <Modal
+        modalRef={modalRef}
+        title="Are you sure you want to delete this event?"
+        modalId="remove-objective-modal"
+        onOk={async () => onDeleteEvent(id)}
+        okButtonText="Delete"
+        okButtonAriaLabel="This button will delete the event"
```

Comment:
You can just say "delete" "delete event" here, otherwise it reads off "BUTTON: THIS BUTTON WILL DELETE THE EVENT" or something equally obnoxious

Reply:
Updated to "delete event".